package.json

```diff
@@ -59,13 +49,38 @@
     "mocha": "^8.1.3",
     "prettier": "^3.1.1",
     "tar": "^6.1.14",
-    "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
-  "dependencies": {
-    "bindings": "^1.5.0",
-    "node-addon-api": "^6.1.0",
-    "node-fetch": "^2.6.1",
-    "node-gyp": "^10.0.1"
-  }
+  "optionalDependencies": {
+    "@jazzer.js/core": "^2.1.0"
```

Comment:
This lib is not compatible with `linux-arm64`.  I attempted to fix it but just ended up fighting a battle that was not worth it.  Its an issue with how the base `C` lib exports artifacts for CMake in `@jazzer/fuzzer`.  I put up an issue in their repo here https://github.com/CodeIntelligenceTesting/jazzer.js/issues/795